db/migrate/20231111175219_change_author_id_to_user_id_in_posts.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeAuthorIdToUserIdInPosts < ActiveRecord::Migration[7.1]
+  def change
+    rename_column :posts, :author_id, :user_id
+  end
+end
```

Comment:
- Please, you should remove this migration file, as you've basically changed your database structure which doesn't align with the ERD diagram from the previous milestone as this shouldn't be changed. Let's kindly update all the changes we've made.